frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -1,25 +1,38 @@
 import React from "react";
+import { merge } from "icepick";
 import { ColorGetter } from "metabase/static-viz/lib/colors";
+import { colors } from "metabase/lib/colors";
 import { XYChart } from "../XYChart";
-import { ChartSettings, ChartStyle, Series } from "../XYChart/types";
+import {
+  ChartSettings,
+  ChartStyle,
+  SeriesWithOneOrLessDimensions,
+  SeriesWithTwoDimensions,
+} from "../XYChart/types";
 import { Colors } from "./types";
 import {
   adjustSettings,
   calculateChartSize,
   getXValuesCount,
 } from "./utils/settings";
+import {
+  getSeriesWithColors,
+  getSeriesWithLegends,
+  removeNoneSeriesFields,
+} from "./utils/series";
 
 interface LineAreaBarChartProps {
-  series: Series[];
-  settings: ChartSettings;
+  multipleSeries: (SeriesWithOneOrLessDimensions | SeriesWithTwoDimensions)[][];
```

Comment:
Wdyt about the following suggestion which may clarify it for further readers:
```
type CardsSeries = Series
type Series = {
  cardName: string;
  column: DatasetColumn;
  breakoutValue?: string;
  ...
}
```
Arrays of arrays are not descriptive, but defining a type allows for giving a clear name. Also, a `breakoutValue` seems like a descriptive enough property: if it is not present, then series is not broken out, otherwise it is